### DISCUSSION TITLE
Application did not respond

### DISCUSSION THREAD

**Discussion by miniseekie** [codetheweb/muse#935] [Q&A]
Hey all, for some reason when typing a command for Muse, half of the time it just replies that the app is not responding - even after resetting the container (I'm running it in unraid through docker). Embeds/previews also don't work at all, even when it _will_ respond to a command and play music. Does anyone have any idea of what could be causing this?

**Comment by codetheweb** [comment]
Sounds like your network might be spotty?

**Reply by miniseekie** [reply]
I don't seem to have any other issues with my network... I do have PiHole running in a separate container, but even when I stop it I still get the issue. I've noticed that when skipping songs, it always works but also always tells me the app didn't respond. Any ideas on anything, in particular, I should fiddle with on my network?

**Reply by codetheweb** [reply]
Hmm, it sounds like your network just has high latency. I believe Discord expects applications to respond within 3 seconds.

**Reply by josipledic** [reply]
Did you ever fix this problem @miniseekie? I have the same issue and it feels very strange to say the least. No other issues with my network either. I'm running it in docker on my home server.
